ElastosBountyProgram/front-end/src/I18N/zh.js

```diff
@@ -555,43 +555,45 @@ const zh = {
+    'landing.businessModel.content.1': '连接所有智能设备(物联网',
     'landing.businessModel.content.2': '和自动驾驶汽车)之间',
     'landing.businessModel.content.3': '点对点网络通信缺失的',
-    'landing.businessModel.content.4': '必要链路的连接。',
-    'landing.businessModel.content.5': '为大型区块链应用程序',
-    'landing.businessModel.content.6': '提供安全的运行环境。',
-    'landing.businessModel.content.7': '为大数据和数字内容',
-    'landing.businessModel.content.8': '在区块链上确认所有权',
-    'landing.businessModel.content.9': '通过在区块链上令牌',
-    'landing.businessModel.content.10': '的交换产生财富。',
+    'landing.businessModel.content.4': '必要链路。',
+    'landing.businessModel.content.5': '',
+    'landing.businessModel.content.6': '为大型区块链应用程序提供',
+    'landing.businessModel.content.7': '安全的运行环境。',
+    'landing.businessModel.content.8': '',
+    'landing.businessModel.content.9': '为大数据和数字内容',
+    'landing.businessModel.content.10': '在区块链上确认所有权。',
     'landing.businessModel.content.11': '',
-    'landing.businessModel.content.12': '',
-    'landing.businessModel.content.13': '',
-    'landing.businessModel.content.14': '使用ELA令牌注册IDs来',
-    'landing.businessModel.content.15': '购买DApps、数字产品',
-    'landing.businessModel.content.16': '和云存储。',
+    'landing.businessModel.content.12': '通过在区块链上交换令牌',
+    'landing.businessModel.content.13': '来产生财富。',
+    'landing.businessModel.content.14': '',
+    'landing.businessModel.content.15': '使用ELA令牌注册IDs来购',
+    'landing.businessModel.content.16': '买DApps、数字产品和云存储。',
     'landing.businessModel.content.17': '',
-    'landing.businessModel.content.18': '通过限制数字资产',
-    'landing.businessModel.content.19': '的固定数量创造稀缺性。',
+    'landing.businessModel.content.18': '通过限制数字资产的固定',
+    'landing.businessModel.content.19': '数量创造稀缺性。',
     'landing.businessModel.content.20': '',
     'landing.businessModel.content.21': '赋予大型区块链应用程序',
     'landing.businessModel.content.22': '能够运行在设备上而不是',
     'landing.businessModel.content.23': '在区块链节点上的能力。',
-    'landing.businessModel.content.24': '提供一个安全的运行环境，',
-    'landing.businessModel.content.25': '免除中间环节，避免分布',
-    'landing.businessModel.content.26': '式拒绝服务攻击和其他',
-    'landing.businessModel.content.27': '网络攻击。',
-    'landing.businessModel.content.28': '通过集成SDK，轻松将',
-    'landing.businessModel.content.29': '现有的移动应用程序',
-    'landing.businessModel.content.30': '移植到Elastos平台。',
-    'landing.businessModel.content.31': '通过多种应用来保持',
-    'landing.businessModel.content.32': '数字内容的使用寿命。',
+    'landing.businessModel.content.24': '',
+    'landing.businessModel.content.25': '提供一个安全的运行环境，',
+    'landing.businessModel.content.26': '免除中间环节，避免分布',
+    'landing.businessModel.content.27': '式拒绝服务攻击和其他',
+    'landing.businessModel.content.28': '网络攻击。',
+    'landing.businessModel.content.29': '',
+    'landing.businessModel.content.30': '通过集成SDK，轻松将',
+    'landing.businessModel.content.31': '现有的移动应用程序',
+    'landing.businessModel.content.32': '移植到Elastos平台上。',
+    'landing.businessModel.content.33': '',
```

Comment:
This wont work, the 33, 34 and 35 keys are not referenced anywhere. 
@wangh615 If you want to improve the formatting of CN text (break lines etc), please report it to Dev as that will require codebase changes and adjusting the EN version too. Ideally we would constrain the translations to the keys exposed though. Thanks!